mopidy/core/playlists.py

```diff
@@ -145,8 +179,14 @@ def delete(self, uri):
 
         uri_scheme = urlparse.urlparse(uri).scheme
         backend = self.backends.with_playlists.get(uri_scheme, None)
-        if backend:
+        if not backend:
+            return
+
+        with _backend_error_handling(backend):
             backend.playlists.delete(uri).get()
+            # TODO: emit playlist changed?
```

Comment:
I think we should add more playlist events as part of a working RW playlist implementation.